jest.config.js

```diff
@@ -54,7 +47,10 @@ module.exports = {
 		'\\.(scss)$': '<rootDir>/' + configPath + '/__mocks__/styleMock.js',
 		'\\.(jpg|jpeg|png|gif|eot|otf|webp|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
 			'<rootDir>/' + configPath + '/__mocks__/fileMock.js',
-		...transpiledPackages,
+		[ `@wordpress\\/(${ transpiledPackageNames.join(
+			'|'
+		) })$` ]: '<rootDir>/gutenberg/packages/$1/src',
+		'test/helpers$': '<rootDir>/' + configPath + '/helpers.js',
```

Comment:
This entry is the same one as what we have in the Jest configuration in Gutenberg:
https://github.com/WordPress/gutenberg/blob/b58fbb3c4456967ff4d8d8df184ef98ce83f3efb/test/native/jest.config.js#L58

Reply:
It took a while to understand what was going on, but I think I understand it now 😄. I then tried to see if this would work:
```
moduleNameMapper: {
   [ `@wordpress\\/(.*)` ]: '<rootDir>/gutenberg/packages/$1/src',
},
```
But I get errors like `Could not locate module @wordpress/react-native-bridge mapped as`, which seem to be because not all packages contain `src/index.js`.
I thought to try using `modulePathIgnorePatterns` to exclude non-conforming packages (assuming they're mapped somewhere else):
```
modulePathIgnorePatterns: [
   '<rootDir>/gutenberg/packages/[^/]*/(?!src)/index.js',
],
```
But either the regex wasn't working or my understanding of how this mapping works is a bit flawed still, or both ;) 
Anyway, this all looks great, I was just doing some exploring.